carma_cooperative_perception/config/cp_multiple_object_tracker_node.yaml

```diff
@@ -1,3 +1,3 @@
-execution_frequency_hz: 20.0
+execution_frequency_hz: 10.0
```

Comment:
Previously this was 20Hz because it needed to process 2 sources (SDSM and lidar) at 10Hz. Which is wrong because subscription queue was set to 1 in all nodes in the pipeline which is too low (as you will see in later files). By increasing the queue size, the node will be able to store multiple source data before processing them at constant 10Hz, which the local world model updates should be.

Reply:
It also makes the PSM flicker if it is at 20Hz because all other message types such as BSM and PSM in motion_computation syncs to external object's rate and resets its internal lists. So if external object is at 20Hz, on rviz, it will look like every other PSM will be published because it is 10Hz.